bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/commands/client/SimpleTestCommand.java

```diff
@@ -61,8 +65,6 @@ public static class Flags extends CliFlags {
         private int ackQuorumSize = 2;
         @Parameter(names = { "-n", "--num-entries" }, description = "Entries to write (default 100)")
         private int numEntries = 100;
-        @Parameter(names = { "-c", "--clean-up" }, description = "Clean up ledger created after simple test")
```

Comment:
It is an easily way to generate a ledger. I am not sure if it should be deleted automatically. If we delete it automatically, the only way to get a ledger is writing a program, it looks very inconvenient.

Reply:
This command is for testing the bookkeeper to create a ledger and read the ledger. If already cover the user's case.
If users want to test by themselves, they should create a new one and read it, not support the ledger generated by SimpleTestCommand.